blocks/api/source.js

```diff
@@ -1,12 +1,6 @@
 /**
- * WordPress dependencies
- */
-import { createElement } from '@wordpress/element';
-
-/**
  * External dependencies
  */
-import { nodeListToReact, nodeToReact } from 'dom-react';
```

Comment:
`dom-react` is no longer used and should be dropped as a dependency (both from `package.json` and updating `package-lock.json` accordingly).